gn3/llms/client.py

```diff
@@ -80,141 +69,59 @@ def __init__(self, account, api_key, version="v3", timeout=30, total_retries=5,
         self.mount("https://", adapter)
         self.mount("http://", adapter)
 
-    @staticmethod
-    def format_bibliography_info(bib_info):
-
-        if isinstance(bib_info, str):
-            # Remove '.txt'
-            bib_info = bib_info.removesuffix('.txt')
-        elif isinstance(bib_info, dict):
-            # Format string bibliography information
-            bib_info = "{0}.{1}.{2}.{3} ".format(bib_info.get('author', ''),
-                                                 bib_info.get('title', ''),
-                                                 bib_info.get('year', ''),
-                                                 bib_info.get('doi', ''))
-        return bib_info
-
-    @staticmethod
-    def ask_the_documents(extend_url, my_auth):
+    def get_answer_using_task_id(self, extend_url, my_auth):
+        """call this method with task id to fetch response"""
         try:
-            response = requests.post(
-                base_url + extend_url, data={}, headers=my_auth)
+            response = requests.get(
+               self.answer_url + extend_url, data={}, headers=my_auth)
```

Comment:
can we use urljoin e.g. `urljoin(self.answer_url, extended_url)` incase we have trailing and leading `/`

Reply:
The entire function is from some older code no longer in use :: Removed.
https://github.com/genenetwork/genenetwork3/pull/188/commits/41c352926ca178e65d6c948fdf93b0f987e2878a